roles/postgres-infra/defaults/main.yml

```diff
@@ -1,6 +1,3 @@
 ---
-# defaults file for postgres-infra
-
-postgres_operator_package: "{{ infrastructure.args.deployment.postgres_operator_package | default('postgres-operator') }}"
-#olm_catalog: "{{ infrastructure.args.deployment.olm_catalog | default('operatorhubio-catalog') }}"
-#olm_namespace: "{{ infrastructure.args.deployment.olm_namespace | default('olm') }}"
+postgres.storagesize: 10Gi
```

Comment:
I don't think we can reference vars like this? 

why not just do:
```
postgres:
  storagesize: 10Gi
  version: 10.4
```

Can we also s/version/tag ?